ui/src/router/components/form/components/enricher_config/typeOptions.js

```diff
@@ -1,21 +1,31 @@
-export const enricherTypeOptions = [
-  {
-    value: "nop",
-    inputDisplay: "None",
-    description:
-      "No enrichment gonna happen. Original request will be sent to configured routes",
-  },
-  {
-    value: "docker",
-    inputDisplay: "Docker",
-    description:
-      "Turing will deploy specified image as a pre-processor and will send original request to it for enrichment",
-  },
-  {
-    value: "external",
-    inputDisplay: "External (Coming Soon)",
-    description:
-      "Turing will send original request to the external URL for enrichment",
-    disabled: true,
-  },
-];
+export const enricherTypeOptions = {
+  HTTP_JSON: [
+    {
+      value: "nop",
+      inputDisplay: "None",
+      description:
+        "No enrichment gonna happen. Original request will be sent to configured routes",
+    },
+    {
+      value: "docker",
+      inputDisplay: "Docker",
+      description:
+        "Turing will deploy specified image as a pre-processor and will send original request to it for enrichment",
```

Comment:
nit: `deploy the specified image as a pre-processor and send the original request to it...`